es/util.go

```diff
@@ -79,11 +79,35 @@ func normalizeDestination(tpl map[string]interface{}) {
 }
 
 func normalizeMonitor(tpl map[string]interface{}) {
+	if triggers, ok := tpl["triggers"].([]interface{}); ok {
+		normalizeMonitorTriggers(triggers)
+	}
+
+	delete(tpl, "id")
```

Comment:
Although this won't change after applies, it makes sense to remove as it's not known before creating the resource so can't be specified by the user. 👍